pkg/gofr/gofr.go

```diff
@@ -250,8 +250,28 @@ func (a *App) Logger() logging.Logger {
 
 // SubCommand adds a sub-command to the CLI application.
 // Can be used to create commands like "kubectl get" or "kubectl get ingress".
-func (a *App) SubCommand(pattern string, handler Handler) {
-	a.cmd.addRoute(pattern, handler)
+func (a *App) SubCommand(pattern string, handler Handler, description ...string) {
+	var descParts []string
+	help := ""
+	// Extract help information from the description and construct the new description
+	for _, d := range description {
+		if strings.HasPrefix(d, "-h ") || strings.HasPrefix(d, "--help ") {
```

Comment:
I dont think that having prefix `-h` or `--help` is required to differentiate between description and help text. We can take the first string to be description and then the rest others combine to form the helper text. If only one string is provided only description is printed else both description and helper text are printed.